recipes/sdl2/all/conanfile.py

```diff
@@ -184,6 +184,10 @@ def build(self):
         tools.replace_in_file(os.path.join(self._source_subfolder, "CMakeLists.txt"),
                               "if(NOT (WINDOWS OR CYGWIN))",
                               "if(NOT (WINDOWS OR CYGWIN OR MINGW))")
+        tools.replace_in_file(os.path.join(self._source_subfolder, "CMakeLists.txt"),
```

Comment:
For which versions is this useful? Maybe make it explicit with a `if version ...`?

Reply:
All versions. It was a bug in 2.0.12 that the recipe did not use iconv even if the option was enbabled